### PR TITLE
Remove some deprecated items

### PR DESCRIPTION
**Pull request by emberian**
These have been deprecated for some time. This is a:

[breaking-change]

### CODE REVIEW

File: src/libstd/sys/unix/fs2.rs [hunk 1]
```diff
@@ -69,9 +69,11 @@ impl FileAttr {
         FilePermissions { mode: (self.stat.st_mode as mode_t) & 0o777 }
     }
 
+    #[allow(dead_code)]
```
Comment (cmr) [inline src/libstd/sys/unix/fs2.rs]: (I figure these will come in handy in the future)

File: src/libstd/fs.rs [hunk 2]
```diff
@@ -575,38 +567,6 @@ impl Metadata {
...
     }
-
-    /// Returns the most recent access time for a file.
-    ///
-    /// The return value is in milliseconds since the epoch.
-    #[unstable(feature = "fs_time",
-               reason = "the return type of u64 is not quite appropriate for \
-                         this method and may change if the standard library \
-                         gains a type to represent a moment in time")]
-    #[deprecated(since = "1.1.0",
```
Comment (sfackler) [inline src/libstd/fs.rs]: These are only deprecated in 1.1. Are we going to back port the PR that deprecated this?

Comment (cmr) [inline src/libstd/fs.rs]: I'd hope so! We're still free to break 1.1.

**Comment by alexcrichton** [comment]
Thanks for this! At this point though we're not removing any deprecated stable functionality (we've already cleaned out what was intended to go), but in general removing deprecated unstable items is fine. I'd leave the deprecated unicode stuff for now as it moved out somewhat recently, however.

**Comment by alexcrichton** [comment]
Now that 1.0 has been released I believe that this is now a breaking change, so closing.

**Pull request closed without merging**